### PR TITLE
i really need to work on tests

### PR DESCRIPTION
**Pull request by kyleschnirring**
really need to work on tests. not sure if emitters are async???

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+'use strict';
```
Comment (sumtype) [inline gulpfile.js]: Awesome!

File: package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "url": "git+https://github.com/kyleschnirring/lab-03-fs-async.git"
+  },
+  "author": "kyle schnirring",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/kyleschnirring/lab-03-fs-async/issues"
+  },
+  "homepage": "https://github.com/kyleschnirring/lab-03-fs-async#readme",
+  "devDependencies": {
+    "chai": "^3.5.0",
```
Comment (sumtype) [inline package.json]: You should add 'gulp', 'gulp-eslint', and 'gulp-mocha' to these devDependencies.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (sumtype) [inline package.json]: Looks good!  One minor issue, you seemed to forget a few devDependencies, see comment below.

File: lab-kyle/test/test-async.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+'use strict';
```
Comment (sumtype) [inline lab-kyle/test/test-async.js]: Delete this file if it doesn't do anything.

File: lab-kyle/lib/fs-function.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const fs = require('fs');
+
+exports.bufferfiles = function(textfile, callback) {
+  fs.readFile(textfile, (err, data) => {
+    if (err) {
+      console.err;
```
Comment (sumtype) [inline lab-kyle/lib/fs-function.js]: The node console object does not contain an err property.

File: lab-kyle/gulpfile.js [hunk 6]
```diff
@@ -0,0 +1,65 @@
+'use strict';
```
Comment (sumtype) [inline lab-kyle/gulpfile.js]: Great job!

File: lab-kyle/package.json [hunk 7]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (sumtype) [inline lab-kyle/package.json]: Excellent!

File: package.json [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
```
Comment (sumtype) [inline package.json]: You should delete this file.

File: lab-kyle/test/test-fs-function.js [hunk 9]
```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const read = require(__dirname + '/../lib/fs-function');
+const expect = require('chai').expect;
+
+describe('testing module lib/fs-function.js', function() {
+  describe('testing function bufferfiles()', function() {
+    it('should return the first 8 bits of a text file in hex ', function(done){
+      read.bufferfiles(__dirname + '/../data/one.txt', function(err, data){
+        expect(err).to.equal(err);
```
Comment (sumtype) [inline lab-kyle/test/test-fs-function.js]: This is essentially saying expect(true).to.equal(true);  I suggest you add a more meaningful test assertion, like since the data file exists expect(null).to.equal(err) or something like that :)

File: lab-kyle/test/test-fs-function.js [hunk 10]
```diff
@@ -0,0 +1,16 @@
+'use strict';
```
Comment (sumtype) [inline lab-kyle/test/test-fs-function.js]: This is a good unit test :), since the assignment involves reading data and returning multiple files' data in series I suggest you add an additional integration test to make sure your bufferfiles function succeeds in that regard as well.

File: lab-kyle/index.js [hunk 11]
```diff
@@ -0,0 +1,31 @@
+'use strict';
```
Comment (sumtype) [inline lab-kyle/index.js]: This is a good start but unfortunately this method does not guarantee that the files will be logged in order.

File: lab-kyle/lib/fs-function.js [hunk 12]
```diff
@@ -0,0 +1,14 @@
+'use strict';
```
Comment (sumtype) [inline lab-kyle/lib/fs-function.js]: This is a good start but there is an issue with the way you're console logging your error, see my comment below.
